lib/src/main/java/com/orange/ods/compose/component/button/OdsIconToggleButtonsRow.kt

```diff
@@ -184,17 +199,16 @@ object OdsIconToggleButtonsRow {
 
 @UiModePreviews.Default
 @Composable
-private fun PreviewOdsIconToggleButtonsGroupRow() = Preview {
-    val icons = listOf(
-        OdsIconToggleButtonsRow.Icon(painterResource(id = android.R.drawable.ic_dialog_dialer), "Today"),
-        OdsIconToggleButtonsRow.Icon(painterResource(id = android.R.drawable.ic_dialog_email), "Day"),
-        OdsIconToggleButtonsRow.Icon(painterResource(id = android.R.drawable.ic_dialog_alert), "Month", false)
+private fun PreviewOdsIconToggleButtonsRow() = Preview {
```

Comment:
Icon buttons are bigger than previously I think and it's also the same thing in the preview.
<img width="277" alt="Capture d’écran 2024-02-15 à 17 21 23" src="https://github.com/Orange-OpenSource/ods-android/assets/17410558/0fc02590-1a97-45fe-9215-c72792000d31">